db/migrate/20150202094307_add_range_to_trend_counters.rb

```diff
@@ -0,0 +1,22 @@
+class AddRangeToTrendCounters < ActiveRecord::Migration
+  def change
+    add_column :trend_counters, :interval_start, :datetime
+    add_column :trend_counters, :interval_end, :datetime
+
+    trend_lines = TrendCounter.order(:trend_id).group_by {|counter| counter.trend_id}
```

Comment:
At this point there shouldn't be too much records, but I will ensure it by running the `rake trends:reduce` task as a part of the migration process.
This task is idempotent, so there is no problem running it even if it was executed manually before running the migration.